src/data/equiv/basic.lean

```diff
@@ -187,6 +187,8 @@ e.left_inv x
 @[simp] lemma symm_trans_apply (f : α ≃ β) (g : β ≃ γ) (a : γ) :
   (f.trans g).symm a = f.symm (g.symm a) := rfl
 
+@[simp] theorem symm_symm_apply (f : α ≃ β) (b : α) : f.symm.symm b = f b := rfl
```

Comment:
Don't we already have a simp lemma that `f.symm.symm = f`? Surely that fires first?

Reply:
That's a `simp` lemma, but not a `dsimp` lemma, which is why I needed this, how do you suggest to fix the apparent lint issue?

Reply:
I don't know if you can. Can you try adding #lint locally on the line below this theorem to see if the linter complains?

Reply:
Unfortunately, the linting indeed fails.
bors r-

Reply:
Do you have a suggested solution?

Reply:
Hmm, changing the priority doesn't seem to work, but getting rid of the `@[simp]` tag on `equiv.symm_symm` does. Not sure if that breaks things outside this file though.

Reply:
I asked [on Zulip](https://leanprover.zulipchat.com/#narrow/stream/113488-general/topic/Simp.20normal.20form/near/219659256) and I think just disabling the linter for now is best:

 ```suggestion
 -- The `simp` attribute is needed to make this a `dsimp` lemma.
 -- `simp` will always rewrite with `equiv.symm_symm` before this has a chance to fire.
@[simp, nolint simp_nf] theorem symm_symm_apply (f : α ≃ β) (b : α) : f.symm.symm b = f b := rfl
```

bors d+

Reply:
Sounds good to me, thanks